package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-time-select",
+  "version": "1.0.0",
+  "description": "Component to generate a dropdown list populated with a configurable time range",
+  "main": "index.js",
+  "scripts": {
+    "start": "ulimit -n 9999 && watchify -t babelify doc/example.js -o doc/example-built.js",
+    "test": "mocha --compilers js:babel/register --require intl test/test*",
```

Comment:
I'd prefer not to mix up those concerns. Don't mind adding a `make ci` to combine them, but I don't think `npm test` should do things unrelated to testing.